### PR TITLE
second cleanup

### PR DESCRIPTION
**Pull request by mimodi9271**

### CODE REVIEW

File: DB/Queries/categories.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
-import { pool } from "../databaseConnection.js"
 
 
-const getCategories = async (website_id) => {
+const getCategories = async (website_id , pool) => {
```
Comment (creativefctr) [inline DB/Queries/categories.js:3]: @mimodi9271 "pool" is a very generic name, use a better, more specific name

File: DB/Queries/tags.js [hunk 2]
```diff
@@ -1,6 +1,5 @@
-import { pool } from "../databaseConnection.js"
 
-const getTags = async (website_id) => {
+const getTags = async (website_id , pool) => {
```
Comment (creativefctr) [inline DB/Queries/tags.js:2]: @mimodi9271 same here and everywhere

File: Utilities/getWebsitePagesURL.js [hunk 3]
```diff
@@ -1,8 +1,7 @@
 import getPageURl from "./getPageURl.js";
-import getPagePost from "./getPagePostURl.js";
 import getPagePostURl from "./getPagePostURl.js";
 
-const getAllPage = async(pages , page_posts , domain) => {
+const getWebsitePagesURL = async(pages , page_posts , domain) => {
     const AllPagesList = [];
```
Comment (creativefctr) [inline Utilities/getWebsitePagesURL.js:5]: @mimodi9271 variable names do not start with capital letters

File: backup/htmlBackup/htmlBackup.js [hunk 4]
```diff
@@ -1,16 +1,14 @@
...
+import backupFinishEmail from '../../emails/backupFinishEmail.js';
 
-
-const initializeBullMq =async (pages , page_posts , redisConnection , domain , email) => {
+const htmlBackup =async (pages , page_posts , redisConnection , domain , email) => {
     
-    let AllPagesList = await getAllPage(pages , page_posts , domain);
-
-  
+    let AllPagesList = await getWebsitePagesURL(pages , page_posts , domain);
```
Comment (creativefctr) [inline backup/htmlBackup/htmlBackup.js:11]: @mimodi9271 same here

File: backup/wordpressBackup/wordpressBackup.js [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+  }
+
+  
+  // add users to xml
+  addUsers(generator , users)
+  
+
+  // add categories to xml
+  addCategories(generator , categories)
+  console.log(categories.length)
```
Comment (creativefctr) [inline backup/wordpressBackup/wordpressBackup.js:50]: @mimodi9271 remove console.log

File: emails/backupFinishEmail.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+import transporter from '../config/mailConf.js';
+
+
+const backupFinishEmail = async (email) => {
+
+    const mailData = {
+        from: process.env.mymailname_mailer,
+        to: email,
+        subject: 'Sending Email using Node.js',
+        text: 'That was not easy!',
```
Comment (creativefctr) [inline emails/backupFinishEmail.js:11]: @mimodi9271 add html email template

File: emails/backupFinishEmail.js [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+    };
+
+    try {
+        const res = await transporter.sendMail(mailData);
+        if(res) console.log("the url has send ..........")
```
Comment (creativefctr) [inline emails/backupFinishEmail.js:16]: @mimodi9271 typo. please fix

File: DB/Queries/categories.js (hunk 1 above)
Comment (mimodi9271) [reply DB/Queries/categories.js:3]: @creativefctr done

File: DB/Queries/tags.js (hunk 2 above)
Comment (mimodi9271) [reply DB/Queries/tags.js:2]: @creativefctr 
done

File: Utilities/getWebsitePagesURL.js (hunk 3 above)
Comment (mimodi9271) [reply Utilities/getWebsitePagesURL.js:5]: @creativefctr 
done

File: backup/htmlBackup/htmlBackup.js (hunk 4 above)
Comment (mimodi9271) [reply backup/htmlBackup/htmlBackup.js:11]: @creativefctr 
done

File: backup/wordpressBackup/wordpressBackup.js (hunk 5 above)
Comment (mimodi9271) [reply backup/wordpressBackup/wordpressBackup.js:50]: @creativefctr 
done

**Pull request merged**